examples/wasm-demo/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "wasm-demo"
+description = "A demo using cairo-vm in a WASM environment"
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+readme.workspace = true
+keywords.workspace = true
```

Comment:
It may be a good idea to set `publish = false`.

Reply:
Yeah 😅